### ISSUE TITLE
command: docker compose is not found. (self-hosted)

### ISSUE THREAD

**Issue by iamyoki** [actions/checkout#640]
![Xnip2021-11-19_11-57-50](https://user-images.githubusercontent.com/74389358/142562664-63557967-a436-49a1-a98c-107c8192c21a.png)

As you can see, same codes `docker compose --help` before and after use checkout. I ran this on my hosted server machine which have installed newest docker, and `docker compose` command is locally fine, but somehow it cannot find after using action/checkout.

**Comment by iamyoki** [comment]
This is my action file
```yml
name: 🚀 Realase On Host 📡
on:
  push:
    branches:
      - main

jobs:
  build:
    runs-on: self-hosted
    steps:
      - name: Before checkout uses, docker compose is fine
        run: docker compose --help
      - uses: actions/checkout@v2
      - name: After checkout uses, docker compose is not found
        run: docker compose --help
```

**Comment by iamyoki** [comment]
Sorry about this report, it's my host server problem....
